src/App.js

```diff
@@ -1,23 +1,17 @@
-import React, { Component } from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-
+import React, {
+  Component
+} from 'react';
 import './App.css';
-import Home from './components/Home/Home';
-import OrderReview from './components/OrderReview/OrderReview';
-import NotFound from './components/NotFound/NotFound';
+
+import Header from './components/Header/header';
```

Comment:
A base filename should exactly match the name of its default export. [More info](https://github.com/airbnb/javascript#naming--filename-matches-export)